conda/gateways/disk/delete.py

```diff
@@ -27,35 +27,39 @@
 
 def rmtree(path, *args, **kwargs):
     # subprocessing to delete large folders can be quite a bit faster
+    path = normpath(path)
     if on_win:
         try:
-            check_call('rd /s /q "{}"'.format(path), shell=True)
-        except CalledProcessError as e:
-            if e.returncode == 5:
-                # skip permission errors
-                log.warn("")
-                pass
-            else:
-                raise
+            # the fastest way seems to be using DEL to recursively delete files
+            # https://www.ghacks.net/2017/07/18/how-to-delete-large-folders-in-windows-super-fast/
+            # However, this is not entirely safe, as it can end up following symlinks to folders
+            # https://superuser.com/a/306618/184799
+            # so, we stick with the slower, but hopefully safer way.  Maybe if we figured out how
+            #    to scan for any possible symlinks, we could do the faster way.
+            # out = check_output('DEL /F/Q/S *.* > NUL 2> NUL'.format(path), shell=True,
+            #                    stderr=STDOUT, cwd=path)
+            out = check_output('RD /S /Q "{}" > NUL 2> NUL'.format(path), shell=True,
+                               stderr=STDOUT)
+        except CalledProcessError:
+            log.debug("removing dir contents the fast way failed.  Output was: {}".format(out))
```

Comment:
I could be wrong, but, if the deletion failed and it was not due to permissions error, the new code flow will engulf it, whereas the previous one raised it. Is that an acceptable change in behavior?